### ISSUE TITLE
network test not getting support?

### ISSUE THREAD

**Issue by ssskip** [opentok/accelerator-core-js#64]
- subscribe own stream

```javascript
if (streamMap[streamId]) {
          // Are we already subscribing to the stream?
          var _state$all = state.all(),
              subscribers = _state$all.subscribers;
          resolve(subscribers[type][streamMap[streamId]]);
} else {
....
}
```

- subscriber container not accept dom object

``` javascript
const container = dom.query(streamContainers('subscriber', type, connectionData, stream));

```

**Comment by adrice727** [comment]
@ssskip I can make an update to address both issues.  I would prefer not to make a general exception for subscribing to our own stream, so I may just let `core.subscribe` accept a third boolean parameter indicating that we are subscribing as part of a network test.

I'll also change `dom.query` to `dom.element` so that an element can be passed.

**Comment by adrice727** [comment]
Fixed in `2.0.14`.  See https://github.com/opentok/accelerator-core-js/pull/65.
